pkg/kubectl/cmd/apply.go

```diff
@@ -218,7 +223,12 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out io.Writer, options *App
 		if !dryRun {
 			overwrite := cmdutil.GetFlagBool(cmd, "overwrite")
 			helper := resource.NewHelper(info.Client, info.Mapping)
-			patcher := NewPatcher(encoder, decoder, info.Mapping, helper, overwrite)
+			patcher := NewPatcher(encoder, decoder, info.Mapping, helper, overwrite,
```

Comment:
@ymqytw I would like to use struct composite literal here, seems the `NewPatcher` has too many arguments.